src/views/Editor/index.js

```diff
@@ -163,47 +154,52 @@ var EditorPage = React.createClass({
         }
     },
 
-    getSaveButtons: function() {
+    getSaveButtons: function(disabled=false) {
         let buttonStyle = {
             height: '64px',
             margin: '0 10px'
         }
         let publicationStatus = _.get(this.props, 'editor.values.publication_status')
-        let labelText = this.props.editor.isSending ? "Julkaistaan tapahtumaa" : "Julkaise tapahtuma"
+        // if publication status field is not present, the user is not logged in and the event is public
+        let eventExists = this.props.params.action === 'update' &&
+            (publicationStatus ? (publicationStatus === constants.PUBLICATION_STATUS.PUBLIC) : true)
```

Comment:
Perhaps use regular booleans instead of ternary operator here?

I assume this part means: eventExists is action is 'update' and publicationStatus is PUBLIC

Reply:
Nooooope, eventExists means action is 'update' and publicationStatus is PUBLIC *or* absent :P

Reply:
Ok, well, more the reason to perhaps state them bools differently?